docs/articles/documentation/guides/basic-guides/run-tests.md

```diff
@@ -0,0 +1,633 @@
+
+```sh
+testcafe chrome tests/test.js -L
+```
+
+In the API, create a [live mode runner](../../reference/api/livemoderunner/README.md) with the [testcafe.createLiveModeRunner](../../reference/api/testcafe/createlivemoderunner.md) function and use it instead of a [regular test runner](../../reference/api/runner/README.md).
+
+```js
+const createTestCafe = require('testcafe');
+let testcafe         = null;
+
+createTestCafe('localhost', 1337, 1338)
+    .then(tc => {
+        testcafe         = tc;
+        const liveRunner = testcafe.createLiveModeRunner();
+
+        return liveRunner
+            .src('tests/test.js')
+            .browsers('chrome')
+            .run();
+    })
+    .then(() => {
+        testcafe.close();
+    });
+```
+
+### How Live Mode Works
+
+When you run tests with live mode enabled, TestCafe opens the browsers, runs the tests, shows the reports, and waits for further actions.
+
+Then TestCafe starts watching for changes in the test files and all files referenced in them (like page objects or helper modules). Once you make changes in any of those files and save them, TestCafe immediately reruns the tests.
+
+When the tests are done, browsers stay on the last opened page so you can work with it and explore it with the browser's developer tools.
+
+You can use live mode with any browsers: local, remote, mobile or headless.
+
+> Important! Live mode is designed to work with tests locally. Do not use it in CI systems.
+
+**Tip:** use the [only](test-organization.md#skip-tests) function to work with a single test.
+
+### Console Shortcuts in Live Mode
+
+* `Ctrl+S` - stops the current test run;
+* `Ctrl+R` - restarts the current test run;
+* `Ctrl+W` - turns off/on the file watcher;
+* `Ctrl+C` - closes opened browsers and terminates the process.
+
+## Quarantine Mode
+
+The quarantine mode is designed to isolate *non-deterministic* tests (that is, tests that pass and fail without any apparent reason) from the other tests.
+
+When the quarantine mode is enabled, tests run according to the following logic:
+
+1. A test runs at the first time. If it passes, TestCafe proceeds to the next test.
+2. If the test fails, it runs again until it passes or fails three times.
+3. The most frequent outcome is recorded as the test result.
+4. If the test result differs between test runs, the test is marked as unstable.
+
+Use the [-q (--quarantine-mode)](../../reference/command-line-interface.md#-q---quarantine-mode) command line flag or the `quarantineMode` option in the [runner.run](../../reference/api/runner/run.md) method to enable quarantine mode:
+
+```sh
+testcafe chrome ./tests/ -q
+```
+
+```js
+await runner
+    .browsers('chrome')
+    .src('./tests/')
+    .run({ quarantineMode: true });
+```
+
+> Note that quarantine mode increases the test task's duration if you enable it on your test machine because failed tests are executed three to five times.
```

Comment:
Moreover, I'm not sure I understand now what `if you enable it on your test machine` means.
I think it should be: *Note that quarantine mode increases the test task's duration because failed tests are executed three to five times.*